rust/core-lib/src/tabs.rs

```diff
@@ -893,25 +893,25 @@ impl BufferIdentifier {
 // =============== Fuchsia-specific synchronization plumbing
 // We can't move this elsewhere since it requires access to private fields
```

Comment:
Can you make the fields `pub(crate)` or some other visibility that would allow you to move more of the cfg'd code into its own module?

Reply:
This is going to get refactored; the idea is that we'll define an interface for multi-device sync that can then get provisioned, and the platform-specific implementation for that will live in the binary wrapper. We could move things around a bit before then, but I'm not feeling strongly motivated.

Reply:
SGTM.